File: fixtures/adapter-java/basic/src/main/java/com/example/Elsewhere.java
package com.example;

// UserService is declared in another file. Tier 2 does not resolve across
// files, so this reference records no edge — and `status` reports the tier so
// nobody mistakes that silence for "nothing uses UserService".
public class Elsewhere {
    public void run() {
        UserService service = new UserService();
        service.handle();
    }
}

File: fixtures/adapter-ruby/basic/elsewhere.rb
# UserService is declared in another file. Tier 2 does not resolve across
# files, so this reference records no cross-file edge — and `status` reports
# the tier so nobody mistakes that silence for "nothing uses UserService".
require_relative "user_service"

class Elsewhere
  def run
    service = UserService.new
    service.handle
  end
end
